template/bash_profile.template.bash

```diff
@@ -31,6 +31,11 @@ export SCM_CHECK=true
 # Will otherwise fall back on $HOSTNAME.
 #export SHORT_HOSTNAME=$(hostname -s)
 
+# Set Xterm/screen/Tmux title with only a short username.
+# Uncomment this (or set SHORT_USER to something else),
+# Will otherwise fall back on $USER.
+#export SHORT_USER=$(echo $USER | sed -r 's/^(.{3}).{3,}(.{3})$/\1..\2/')
```

Comment:
The `sed -r` switch is only supported in Gnu sed, it does not work on the one that comes with macOS, for example. It might be better to move this regular expression to the plugin itself and put it inside of an `if` statement that uses the correct `sed` syntax depending on the OS.

That would mean that you need a FLAG in the profile to enable this functionality - by default (if the flag is not set) it should use the existing behavior.